### ISSUE TITLE
Vlat_file

### ISSUE THREAD

**Issue by solomonsg** [c-h-david/rapid#17]
Dear David,

I start learning and working with rapid and found it very helpful. I have prepared all the input files but i don't know how to merge river networks (.shp) and runoff data to be in Vlat_file. do you have any script or any method to combine these files? in addition, the runoff data i have is surface and baseflow. 

Thanks for your help 

Solomon

**Comment by solomonsg** [comment]
Dear David, 
I used ncks -A to merge the two files but I am getting another error "Fortran runtime error: Cannot open file './rapid_namelist'"


![image](https://user-images.githubusercontent.com/48722659/63627105-f76ca100-c5fd-11e9-8bb8-1c372742d565.png)

**Comment by c-h-david** [comment]
Dear Solomon,
Thank you for your interest in RAPID and welcome to its community of users!  I do all my pre- and post-processing for RAPID using the Python toolbox located at: https://github.com/c-h-david/rrr. You might want to take a look at that in addition to this repository. The "rapid_namelist" error that is shown above is because RAPID looks for a file called "rapid_namelist" at execution. This is where you specify all the variables and file names. You can either use a file with this exact name, or create a symbolic link (a shortcut) between rapid_namelist and whatever file you want to use. 
I hope this helps!
Cedric

**Comment by solomonsg** [comment]
Dear David, 

Thanks and running it as "./rapid  **./**rapid_namelist" helps, but I think NCO and CDO are not helpful to merge runoff and river network as required by RAPID? can you please share the tool you are using to merge the files. this is what I am now
![image](https://user-images.githubusercontent.com/48722659/63786919-d8fef200-c8ea-11e9-9aa6-232541f2dc12.png)

Best regards,

Solomon

**Comment by c-h-david** [comment]
Dear Solomon,
You should look into the two bash scripts at https://github.com/c-h-david/rrr/blob/master/tst/tst_pub_dwnl_David_etal_2019_GRL.sh and https://github.com/c-h-david/rrr/blob/master/tst/tst_pub_repr_David_etal_2019_GRL.sh. The former downloads example data and the latter reproduces all the processing steps involved in a recently-published paper. Specifically, two scripts are used for transforming runoff grids into RAPID inflows: `rrr_cpl_riv_lsm_lnk.py` and `rrr_cpl_riv_lsm_vol.py`. At this point this is probably your best bet. I do not have a users' manual for RRR, but at least this gives you an idea of how the programs are used.
Best,
Cedric

**Comment by solomonsg** [comment]
Hi David,

Thanks for the scripts. I finally manged to create the runoff file but when i run rapid i am not getting the full output. the input runoff file is daily from 1979-2016 and this is what i am getting now

Checked for missing connections between basin studied and rest of domain
--------------------------
Cumulative time for routing only
Rank     :         0, Time     :      0.00
Output data created
--------------------------
type of vector: seq
type of matrix: seqaij
type of KSP   : richardso
type of PC    : ilu
Rank     :         0, Max KSP  :         0
--------------------------

I am using gfortran (7.3.0) and it is F95 not F90. 

I am attaching the namelist and input and output files. 
[Rapid.zip](https://github.com/c-h-david/rapid/files/3677586/Rapid.zip)

Best regards,

Solomon
